src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/GraphQLSupportedTypesTestsBase.cs

```diff
@@ -262,6 +281,26 @@ public async Task InsertIntoTypeColumn(string type, string value)
             await ResetDbStateAsync();
         }
 
+        [DataTestMethod]
+        [DataRow(TIME_TYPE, "\"32:59:59.9999999\"")]
+        [DataRow(TIME_TYPE, "\"22:67:59.9999999\"")]
```

Comment:
add comment for what you are validating. This may address one scenario i mentioned in another comment: including a value representing a time over 24 hours and attempt to have DAB convert it to TimeOnly. SHould also try a negative value.